### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scrollTop' of undefined

### ISSUE THREAD

**Issue by match08** [driftyco/ionic#4841]
ion-content  is  $element.bind('scroll', function (e) {
    
           //support for jQuery event as well
            e = e.originalEvent || e;
            var scrollTop = e.detail.scrollTop;
           
}
detail is ionic 1.2 not working is undefind,on ionic 1.1 ok.

**Comment by Ionitron** [comment]
Greetings @match08!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4841). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4841)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by jgw96** [comment]
Hello! While we appreciate you opening an issue with us i am going to close this issue and ask that you open a new one that has the issue template filled out please. Thanks for using Ionic!
